parsepush.go

```
@@ -134,7 +142,19 @@ func (c *Conn) do() {
 
 		select {
 		case push := <-lr.lines:
-			c.pushHandler(push)
+			var payload Payload
+			if err := json.Unmarshal(push, &payload); err != nil {
+				handleErr(err)
+				break
+			}
+			// Ignore empty "{}" keep-alive response from server.
+			if len(payload.Time) == 0 && len(payload.Data) == 0 {
+				break
+			}
+			if payload.Time > c.LastTime() {
+				ConnLastTime(payload.Time)(c)
```

this is strange and also ignores an error (which obviously never happens). prefer directly `c.lastTime.Store(payload.Time)` instead.